pkg/log/log_test.go

```diff
@@ -83,3 +88,45 @@ func (s *testLogSuite) TestLogLevel(c *C) {
 	c.Assert(L().Check(zap.WarnLevel, "This is a warn log"), NotNil)
 	c.Assert(L().Check(zap.DebugLevel, "This is a debug log"), IsNil)
 }
+
+func captureStdout(f func()) ([]string, error) {
+	r, w, _ := os.Pipe()
+	stdout := os.Stdout
+	os.Stdout = w
+
+	f()
+
+	var buf bytes.Buffer
+	output := make(chan string, 1)
+	errs := make(chan error, 1)
+
+	go func() {
+		_, err := io.Copy(&buf, r)
+		output <- buf.String()
+		errs <- err
+		r.Close()
+	}()
+
+	os.Stdout = stdout
+	w.Close()
+	return strings.Split(<-output, "\n"), <-errs
+}
+
+func (s *testLogSuite) TestInitSlowQueryLogger(c *C) {
+	logLevel := "debug"
+	cfg := &Config{
+		Level: logLevel,
+	}
+	cfg.Adjust()
+
+	output, err := captureStdout(func() {
+		c.Assert(InitLogger(cfg), IsNil)
+		logutil.SlowQueryLogger.Debug("this is test info")
+		appLogger.Debug("this is from applogger")
+		// test there are two different loggers
+		c.Assert(appLogger, Not(Equals), logutil.SlowQueryLogger)
```

Comment:
ineffective check. maybe one crucial field of two logger duplicated will cause problem.